nx.json

```diff
@@ -0,0 +1,37 @@
+{
+  "extends": "nx/presets/npm.json",
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "cs-check",
+          "cs-format",
+          "lint",
+          "test",
+          "test:update",
+          "test:watch",
+          "precommit",
+          "publish-to-npm",
+          "test-coverage",
+          "build:lib",
+          "prepublishOnly",
+          "build:dist",
+          "publish-to-gh-pages"
+        ],
+        "accessToken": "N2UzOTU5MGQtMmM1OS00MTM2LWJjM2QtZDZhM2IxOWYzM2RkfHJlYWQtd3JpdGU="
```

Comment:
Is this a secret that needs to be passed in as a variable? I'm not sure we want to use the cloud... and if we did, we really don't want this secret sitting in a public file